File: trunk/采集端/cis-business/src/main/java/com/sinotao/business/dao/entity/ClinicarCheckItem.java
package com.sinotao.business.dao.entity;

import javax.persistence.Column;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

/**
 * 检查项目信息表
 * @author 佟磊
 */
@Table(name = "t_clinicar_check_item")
public class ClinicarCheckItem {

	/**
	 * 代理主键
	 */
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Integer id;
	
	/**
	 * 删除标记
	 */
	private Boolean deleted;
	
	/**
	 * 登记表ID
	 */
	@Column(name = "check_id")
	private Integer checkId;
	
	/**
	 * 检查项目编号
	 */
	@Column(name = "item_code")
	private String itemCode;
	
	/**
	 * 检查项目名称
	 */
	@Column(name = "item_name")
	private String itemName;

	/**
	 * 科室编号
	 */
	@Column(name = "dpt_code")
	private String dptCode;

	/**
	 * 科室名称
	 */
	@Column(name = "dpt_name")
	private String dptName;
	
	/**
	 * 该项检查是否完成
	 */
	private Boolean completed;
	
	/**
	 * 弃检
	 */
	private Boolean canceled;

	/**
	 * 小结
	 */
	private String summary;
	
	/**
	 * 结论
	 */
	private String conclusion;
	
	/**
	 * 建议
	 */
	private String advice;

	/**
	 * @return the id
	 */
	public Integer getId() {
		return id;
	}

	/**
	 * @param id the id to set
	 */
	public void setId(Integer id) {
		this.id = id;
	}

	/**
	 * @return the deleted
	 */
	public Boolean getDeleted() {
		return deleted;
	}

	/**
	 * @param deleted the deleted to set
	 */
	public void setDeleted(Boolean deleted) {
		this.deleted = deleted;
	}

	/**
	 * @return the checkId
	 */
	public Integer getCheckId() {
		return checkId;
	}

	/**
	 * @param checkId the checkId to set
	 */
	public void setCheckId(Integer checkId) {
		this.checkId = checkId;
	}

	/**
	 * @return the itemCode
	 */
	public String getItemCode() {
		return itemCode;
	}

	/**
	 * @param itemCode the itemCode to set
	 */
	public void setItemCode(String itemCode) {
		this.itemCode = itemCode;
	}

	/**
	 * @return the itemName
	 */
	public String getItemName() {
		return itemName;
	}

	/**
	 * @param itemName the itemName to set
	 */
	public void setItemName(String itemName) {
		this.itemName = itemName;
	}

	/**
	 * @return the dptCode
	 */
	public String getDptCode() {
		return dptCode;
	}

	/**
	 * @param dptCode the dptCode to set
	 */
	public void setDptCode(String dptCode) {
		this.dptCode = dptCode;
	}

	/**
	 * @return the dptName
	 */
	public String getDptName() {
		return dptName;
	}

	/**
	 * @param dptName the dptName to set
	 */
	public void setDptName(String dptName) {
		this.dptName = dptName;
	}

	/**
	 * @return the completed
	 */
	public Boolean getCompleted() {
		return completed;
	}

	/**
	 * @param completed the completed to set
	 */
	public void setCompleted(Boolean completed) {
		this.completed = completed;
	}

	/**
	 * @return the canceled
	 */
	public Boolean getCanceled() {
		return canceled;
	}

	/**
	 * @param canceled the canceled to set
	 */
	public void setCanceled(Boolean canceled) {
		this.canceled = canceled;
	}

	/**
	 * @return the summary
	 */
	public String getSummary() {
		return summary;
	}

	/**
	 * @param summary the summary to set
	 */
	public void setSummary(String summary) {
		this.summary = summary;
	}

	/**
	 * @return the conclusion
	 */
	public String getConclusion() {
		return conclusion;
	}

	/**
	 * @param conclusion the conclusion to set
	 */
	public void setConclusion(String conclusion) {
		this.conclusion = conclusion;
	}

	/**
	 * @return the advice
	 */
	public String getAdvice() {
		return advice;
	}

	/**
	 * @param advice the advice to set
	 */
	public void setAdvice(String advice) {
		this.advice = advice;
	}
}

File: trunk/医院端/cis-model/clinicar/ClinicarCheckItem.cs
using Castle.ActiveRecord;
using System;
using System.Collections.Generic;
using System.Text;

namespace cis_model.clinicar
{
    /// <summary>
    /// 登记检查项目
    /// </summary>
    [ActiveRecord("t_clinicar_check_item")]
    public class ClinicarCheckItem : SCommon.SBase.SBaseModel<ClinicarCheckItem>
    {
        /// <summary>
        /// 主键
        /// </summary>
        [PrimaryKey(PrimaryKeyType.Identity, "id")]
        [SCommon.SAttribute.SModelProperty("id", Display = "主键")]
        public Int32 Id { get; set; }

        /// <summary>
        /// 检查ID（外键）
        /// </summary>
        [Property("check_id")]
        [SCommon.SAttribute.SModelProperty("check_id", Display = "检查ID")]
        public Int32 CheckId { get; set; }

        /// <summary>
        /// 检查项目编号
        /// </summary>
        [Property("item_code")]
        [SCommon.SAttribute.SModelProperty("item_code", Display = "项目编号")]
        public String ItemCode { get; set; }

        /// <summary>
        /// 项目名称
        /// </summary>
        [Property("item_name")]
        [SCommon.SAttribute.SModelProperty("item_name", Display = "项目名称")]
        public String ItemName { get; set; }

        /// <summary>
        /// 科室编号
        /// </summary>
        [Property("dpt_code")]
        [SCommon.SAttribute.SModelProperty("dpt_code", Display = "科室编号")]
        public String DptCode { get; set; }

        /// <summary>
        /// 科室
        /// </summary>
        [Property("dpt_name")]
        [SCommon.SAttribute.SModelProperty("dpt_name", Display = "科室")]
        public String DptName { get; set; }

        /// <summary>
        /// 该项检查是否完成
        /// </summary>
        [Property("completed")]
        [SCommon.SAttribute.SModelProperty("completed", Display = "检查状态")]
        public Boolean Completed { get; set; }

        /// <summary>
        /// 小结
        /// </summary>
        [Property("summary")]
        [SCommon.SAttribute.SModelProperty("summary", Display = "小结")]
        public String Summary { get; set; }

        /// <summary>
        /// 删除标记
        /// </summary>
        [Property("deleted")]
        [SCommon.SAttribute.SModelProperty("deleted", Display = "删除标记")]
        public Boolean Deleted { get; set; }

        /// <summary>
        /// 该项检查是否弃检
        /// </summary>
        [Property("canceled")]
        [SCommon.SAttribute.SModelProperty("canceled", Display = "选中")]
        public Boolean Canceled { get; set; }
    }
}
